### PR TITLE
2024 SC Election Files

### PR DESCRIPTION
**Pull request by creydr**
Adding the initial files required to run the 2024 Steering Committee election.

Before this can be merged, we need the following approvals:

* @knative/steering-committee - confirm that details are correct and acceptable to SC (number of seats, schedule, etc.)
* @knative/steering-committee - approve / nominate @geekygirldawn and me as election officers for this election
* @jberkus - confirm that I haven't mangled any of the things needed by Elekto, and confirm that you are available on the key dates if there are any Elekto problems. Note: I've left you on the elections email alias and in the owners file in case anyone reports Elekto issues via email or you need to merge any urgent changes due to issues with Elekto.

I got the voters list from [knative.devstats.cncf.io](https://knative.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All), by taking everybody who had at least 25 contributions in the last year ([SC: voter-eligibility](https://github.com/knative/community/blob/main/mechanics/SC.md#voter-eligibility))

/hold

### CODE REVIEW

File: elections/2024-SC/README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+| December 17  | Voter exception requests due by 23:59 UTC (4:59pm Pacific) |
+| December 20  | Election Closes by 23:59 UTC (4:59pm Pacific) |
+| December 23  | Announcement of Results |
+
+## Election Officers
+
+In an effort to run a fair and transparent election, the following people
+have been asked by the Steering Committee to run this election:
+
+- [Dawn Foster](https://github.com/geekygirldawn)
```
Comment (creydr) [inline elections/2024-SC/README.md:48]: @jberkus would be be an election officer too?

File: elections/2024-SC/election.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+end_datetime: 2024-12-20 23:59:00
+no_winners: 2
+allow_no_opinion: True
+delete_after: True
+show_candidate_fields:
+  - affiliation
+  - owners
+election_officers:
+  - geekygirldawn
+  - jberkus
```
Comment (creydr) [inline elections/2024-SC/election.yaml:13]: See https://github.com/knative/community/pull/1629/files#r1850119435

**Review by psschwei — COMMENTED** [review]
couple of minor nits

**Comment by geekygirldawn** [comment]
I don't have much time to devote to this, and technically, I'm probably not actually eligible to be an election officer, since I haven't been participating in Knative enough to meet the eligible voter requirement. 

From https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md#eligibility-to-vote 
> the Steering Committee will appoint between one and three Election Officer(s) to administer the election. Elections Officers will be Knative community members in good standing who are eligible to vote ...

Although, I'm guessing that the SC can consider me an exception given my contributions as an election officer in the past :) 

Since the requirement is to have one election officer, we could just make @creydr the election officer, and then @jberkus and I can provide support / answer questions as needed. If you want another election office, I can try to help out, but I don't have a ton of time to devote to it.

File: elections/2024-SC/election.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  - creydr
+eligibility: All Knative contributors with more than 25 contributions in the last 12 months are eligible to vote
```
Comment (creydr) [inline elections/2024-SC/election.yaml:15]: I see two sources for this with different eligibilities:

> Anyone who has at least 25 contributions in the last 12 months is eligible to vote in the SC election

from https://github.com/knative/community/blob/main/mechanics/SC.md#voter-eligibility

vs 

> Anyone who has at least 50 contributions in the last 12 months is eligible to vote in the Steering election

from https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md#eligibility-to-vote

Which one is correct? @knative/steering-committee

Comment (evankanderson) [reply elections/2024-SC/election.yaml:15]: Good question!  Looking at the history, https://github.com/knative/community/pull/1452 and https://github.com/knative/community/pull/1451 seem to have been aimed to adjust the threshold to 25 contributions, so we should correct this in the `STEERING-COMMITTEE.md` file.

Comment (creydr) [reply elections/2024-SC/election.yaml:15]: :+1: Addressed in https://github.com/knative/community/pull/1632

**Comment by dsimansk** [comment]
@creydr feel free to add me as an election officer to help out. I don't plan to submit any candidacy, so hopefully there isn't any conflict of being on the SC roster currently.

**Comment by geekygirldawn** [comment]
@dsimansk - unfortunately, you also aren't eligible as a current SC member:

> Elections Officers will be Knative community members in good standing who are eligible to vote, are not running for Steering in that election, who are not currently part of the Steering Committee and can make a public promise of impartiality. 

However, it doesn't mean that you can't help out. I think that if we make @creydr the official election officer that with @dsimansk helping out and @jberkus and I for questions, I think we should be ok?

**Comment by psschwei** [comment]
> I think that if we make @creydr the official election officer that with @dsimansk helping out and @jberkus and I for questions, I think we should be ok?

+1 works for me

**Comment by creydr** [comment]
> However, it doesn't mean that you can't help out. I think that if we make @creydr the official election officer that with @dsimansk helping out and @jberkus and I for questions, I think we should be ok?

Works for me :+1: 
I can also try to reach out to others if they can support as an election officer before the election starts

**Comment by dprotaso** [comment]
/lgtm

notes: PR is approved just need to drop the hold @creydr

**Pull request merged**